opts.go

```diff
@@ -70,10 +70,11 @@ type options struct {
 	enabledFuncs  []*parser.Function
 	enabledBinops []parser.ItemType
 
-	enableOffset           bool
-	enableAtModifier       bool
-	enableVectorMatching   bool
-	atModifierMaxTimestamp int64
+	enableOffset             bool
+	enableAtModifier         bool
+	enableVectorMatching     bool
+	enableExperimentalPromQL bool
```

Comment:
Can we call it `enableExperimentalPromQLFunctions`? Same as https://prometheus.io/docs/prometheus/latest/feature_flags/#experimental-promql-functions.

It is not just functions but I think it is more accurate to mention that